integration/default_test.go

```diff
@@ -0,0 +1,163 @@
+					settings.Buildpacks.BuildPlan.Online,
+				).
+				Execute(name, filepath.Join("testdata", "default_app"))
+			Expect(err).ToNot(HaveOccurred(), logs.String)
+
+			Expect(logs).To(ContainLines(
+				MatchRegexp(fmt.Sprintf(`%s \d+\.\d+\.\d+`, buildpackInfo.Buildpack.Name)),
+				"  Resolving Visual Studio Debugger version",
+				"    Candidate version sources (in priority order):",
+				"      <unknown> -> \"\"",
+			))
+			Expect(logs).To(ContainLines(
+				MatchRegexp(`    Selected Visual Studio Debugger version \(using <unknown>\): \d+\-\d+\-\d+\-\d+`),
+			))
+			Expect(logs).To(ContainLines(
+				"  Executing build process",
+				MatchRegexp(`    Installing Visual Studio Debugger \d+\-\d+\-\d+\-\d+`),
+				MatchRegexp(`      Completed in \d+\.\d+`),
+			))
+			Expect(logs).To(ContainLines(
+				"  Configuring build environment",
+				MatchRegexp(fmt.Sprintf(`    PATH -> "\$PATH:\/layers\/%s\/vsdbg"`, strings.ReplaceAll(buildpackInfo.Buildpack.ID, "/", "_"))),
+				"",
+				"  Configuring launch environment",
+				MatchRegexp(fmt.Sprintf(`    PATH -> "\$PATH:\/layers\/%s\/vsdbg"`, strings.ReplaceAll(buildpackInfo.Buildpack.ID, "/", "_"))),
+			))
+
+			container, err = docker.Container.Run.
+				WithCommand("which vsdbg").
+				Execute(image.ID)
+			Expect(err).ToNot(HaveOccurred())
+
+			Eventually(func() string {
+				cLogs, err := docker.Container.Logs.Execute(container.ID)
+				Expect(err).NotTo(HaveOccurred())
+				return cLogs.String()
+			}).Should(ContainSubstring(fmt.Sprintf(`/layers/%s/vsdbg`, strings.ReplaceAll(buildpackInfo.Buildpack.ID, "/", "_"))))
+		})
+
+		context("validating SBOM", func() {
+			var (
+				sbomDir string
+			)
+
+			it.Before(func() {
+				var err error
+				sbomDir, err = os.MkdirTemp("", "sbom")
+				Expect(err).NotTo(HaveOccurred())
+				Expect(os.Chmod(sbomDir, os.ModePerm)).To(Succeed())
+			})
+
+			it.After(func() {
+				Expect(docker.Container.Remove.Execute(container.ID)).To(Succeed())
+				Expect(os.RemoveAll(sbomDir)).To(Succeed())
+			})
+
+			it("writes SBOM files to the layer and label metadata", func() {
+				var err error
+				var logs fmt.Stringer
+				image, logs, err = pack.WithNoColor().Build.
+					WithPullPolicy("never").
+					WithBuildpacks(
+						settings.Buildpacks.VSDBG.Online,
+						settings.Buildpacks.BuildPlan.Online,
+					).
+					WithEnv(map[string]string{
+						"BP_LOG_LEVEL": "DEBUG",
+					}).
+					WithSBOMOutputDir(sbomDir).
+					Execute(name, filepath.Join("testdata", "default_app"))
+				Expect(err).ToNot(HaveOccurred(), logs.String)
+
+				container, err = docker.Container.Run.
+					WithCommand("which vsdbg").
+					Execute(image.ID)
+				Expect(err).ToNot(HaveOccurred())
+
+				Eventually(func() string {
+					cLogs, err := docker.Container.Logs.Execute(container.ID)
+					Expect(err).NotTo(HaveOccurred())
+					return cLogs.String()
+				}).Should(ContainSubstring(fmt.Sprintf(`/layers/%s/vsdbg`, strings.ReplaceAll(buildpackInfo.Buildpack.ID, "/", "_"))))
```

Comment:
I just did a quick experiment - turns out `vsdbg --help` spits out some output that could be good to assert against. To me, that gives a slightly better signal that the executable  runs happily in the container.